database/src/config/index.ts

```diff
@@ -6,7 +6,7 @@ dotenv.config()
 const constants = {
   CONFIG_VERSION: {
     DEFAULT: 'DEFAULT',
-    EXPECTED: 'v1.2022-03-18',
+    EXPECTED: 'v7.2022-06-15',
```

Comment:
Here does this come from?

Reply:
We did not change the config of the database for a long time.

Reply:
it was introduced after Wolle had merged the user settings changes with migrationstep 0037